app/assets/stylesheets/components/_card.scss

```diff
@@ -43,3 +43,7 @@
   top: -20px;
   width: 40px;
 }
+
+.card {
```

Comment:
will this card will clash with the existing card class

Reply:
it didn't change any of you cards, because you used the card-trip class